app/components/WorkoutExercise/WorkoutExerciseTrackView/WorkoutExerciseTrackView.tsx

```diff
@@ -35,6 +35,7 @@ const WorkoutExerciseTrackView: React.FC = () => {
       const fromDraft = WorkoutSetModel.create({
         ...draftCopy,
         exercise: stateStore.openedExerciseGuid,
+        date: stateStore.openedDate,
```

Comment:
This could be an issue if the workout stretches into another day.
date: new Date.toISO() or whatever might be better.
Or we could just have a createdAt timestamp